docs/context/development/schema.json

```diff
@@ -332,6 +339,25 @@
             "items": {
               "$ref": "#/definitions/taxonomic_unit"
             }
+          },
+          "apomorphy": {
+            "type": "object",
+            "description": "For a phyloreference using an apomorphy, this field describes the character being used as the apomorphy",
```

Comment:
```suggestion
            "description": "For a phyloreference using an apomorphy, this field describes the phenotype (or trait) being used as the apomorphy",
```

Reply:
An apomorphy is not a character but a character state (or more generally, a phenotype or trait).